Closure_Front_End/src/components/ProgressBox.vue

```diff
@@ -3,14 +3,18 @@
     <br />
     <div class="sub">
       סך הכל:
-      <progress class="progress total" v-bind:value="total" max="100">
+      <progress
+        class="progress total"
+        v-bind:value="sum(1) + sum(2) + sum(3)"
+        max="100"
+      >
         15%
       </progress>
     </div>
 
     <div class="sub">
       חובה:
-      <progress class="progress must" v-bind:value="mustValue" max="80">
+      <progress class="progress must" v-bind:value="sum(1)" max="80">

```

Comment:
`mustValue` is better for code readability, is changing it inevitable?

Reply:
ill change the names so it would be more readable, but i need to keep it binded to a method and not to a variable, otherwise it wont update

Reply:
sure, thanks
please resolve this conversation once they're renamed

Reply:
fixed